.readthedocs.yaml

```diff
@@ -0,0 +1,51 @@
+# .readthedocs.yaml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+# Required
+version: 2
+
+build:
+  os: ubuntu-20.04
+  tools:
+    python: "3.10"
+  apt_packages:
+    - libdbus-1-dev
+    - libgirepository1.0-dev
+    - gir1.2-gtk-3.0
+    - gir1.2-notify-0.7
+    - gir1.2-gudev-1.0
+    - graphviz
+    - imagemagick
+    - libpulse-dev
+    - git
+    - xserver-xephyr
+    - xterm
+    - xvfb
+    - ninja-build
+    - libegl1-mesa-dev
+    - libgles2-mesa-dev
+    - libgbm-dev
+    - libinput-dev
+    - libxkbcommon-dev
+    - libpixman-1-dev
+    - libpciaccess-dev
+    - dbus-x11
+    - libnotify-bin
+    - libegl-dev
+    - libglvnd-dev
+    - libgles-dev
+    - libglvnd-dev
+    - libqt5gui5
+    - libegl1
+    - libegl-mesa0
+# Build documentation in the docs/ directory with Sphinx
+sphinx:
+  configuration: docs/conf.py
+# Optionally build your docs in additional formats such as PDF
+formats:
+- pdf
```

Comment:
I'm not aware of any quotas. Also, our current docs are downloadable as pdf, html and epub so I probably need to add those options.

NB screenshots aren't actually appearing in the pdf which I suspect is because I'm using html code rather than rst for the table. This is because I haven't found another way to get images in a table when iterating in a template.